File: core/src/main/java/ru/hubabank/core/dto/BillDto.java
package ru.hubabank.core.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.util.UUID;

@Data
public class BillDto {

    @Schema(description = "Идентификатор счета")
    private UUID id;

    @Schema(description = "Идентификатор клиента счета")
    private UUID userId;

    @Schema(description = "Баланс счета в копейках")
    private long balance;

    @Schema(description = "Флаг, указывающий на то, что счет закрыт")
    private boolean closed;
}

File: employee-gateway/EmployeeGateway/EmployeeGateway.Common/DTO/Bill/BillDto.cs
namespace EmployeeGateway.Common.DTO.Bill;

public class BillDto
{
    public string Id { get; set; }

    public string UserId { get; set; }

    public long Balance { get; set; }
        
    public string Currency {  get; set; }

    public bool isHidden { get; set; }
}
